docs/general/events.md

```diff
@@ -32,17 +32,74 @@ field semantics, and stability and requirement levels. Other events may be user-
 bespoke user semantics. When an Event name exists in the semantic conventions, its _payload_
 structure and semantics will also be defined.
 
-The following semantic conventions for events are defined:
+## Event definition
 
-* **[General](#general-event-attributes): General semantic attributes that may be used in describing Events.**
-* [Exceptions](/docs/exceptions/exceptions-logs.md): Semantic attributes that may be used in describing exceptions as events.
+<!-- semconv event -->
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`event.name`](../attributes-registry/event.md) | string | Identifies the class / type of event. [1] | `browser.mouse.click`; `device.app.lifecycle` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** Event names are subject to the same rules as [attribute names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common/attribute-naming.md). Notably, event names are namespaced to avoid collisions and provide a clean separation of semantics for events in separate domains like browser, mobile, and kubernetes.
+<!-- endsemconv -->
+
+### General event semantics
+
+* An event MUST have an `event.name` attribute that uniquely identifies the event.
+* It MAY have other [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common#attribute)
+  attributes that provide additional context about the event.
+* It MAY contain a _payload_ (data) that describes the specific details of the
+  named event.
+* The structure of the named event (`event.name`) is unique to the event, this
+  identifies
+  * Whether the event contains a _payload_ (data), and the structure / type of
+    the _payload_ (data).
+  * Whether there are any expected [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common#attribute)
+    attributes that are used to describe the event.
+* All OpenTelemetry defined events SHOULD be defined in the semantic conventions
+  to highlight the expected structure of the event.
+* The _payload_ (data) MAY contain any type supported by the OpenTelemetry data
+  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body)
+  and the semantic conventions will define the expected structure of the _payload_
+  (data) for the event.
+* The _payload_ (data) SHOULD be used to represent the structure of the event.
```

Comment:
Do we still need this sentence? What new information does it convey that's not covered before?

Reply:
We are trying to convey that when defining events it is recommended that you should use the payload to describe the detail of the event vs using a collection of standard attributes.

It's a should because there are some existing legacy cases (exceptions are a good example) where it would probably be too much of a breaking change (or too much additional confusion) to introduce yet another way to represent exceptions that are not directly compatible.